opentelemetry-sdk/src/opentelemetry/sdk/_metrics/measurement_consumer.py

```diff
@@ -0,0 +1,22 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This is done in order to avoid a ciruclar import between instrument.py and
```

Comment:
Since the circular dependency is only caused by type checking, we can just follow this guidance: https://mypy.readthedocs.io/en/stable/runtime_troubles.html#import-cycles. I.e. Put the `MeasurementConsumer` and `SynchronousMeasurementConsumer` in measurement_consumer.py and then in `instrument.py` and use `TYPE_CHECKING` and forward refs

Reply:
Great suggestion, fixed :v: